src/frontends/pytorch/src/op/meshgrid.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_meshgrid(NodeContext& context) {
+    OutputVector inputs{context.get_input_from_visible_context(0)};
```

Comment:
Again, you should check that `context.get_input_from_visible_context(0)` is `Constant` only after that use it as input otherwise you may break graph logic if you add some non-constant node inside the body, that will add not only that node, but all subgraph it belongs to. From the looks of it you do not need to do `get_input_from_visible_context` for input 0, just use regular `context.get_input(0)`

Reply:
Sorry, misunderstood your previous comment, thought that issue was only for input(1). Replaced `context.get_input_from_visible_context(0)` with `context.get_input(0)` as you suggested